pom.xml

```diff
@@ -86,6 +87,12 @@
             <version>1.7.26</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.9.8</version>
+            <scope>provided</scope>
```

Comment:
I think the scope `provided` is wrong here. Unless I am mistaken, eXist-db does not provide this dependency.

The scope should likely be either `compile` if you have a compile time code dependency, or runtime if it is only needed at runtime.